x-pack/legacy/plugins/ml/server/models/data_frame/error_utils.ts

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+// @ts-ignore no declaration file for module
+export { isRequestTimeout } from '../job_service/error_utils';
+import {
+  DataFrameTransformEndpointRequest,
+  DataFrameTransformEndpointResult,
+} from '../../../public/data_frame/pages/transform_management/components/transform_list/common';
+// @ts-ignore no declaration file
+import { MLRequestFailure } from '../../../public/util/ml_error';
```

Comment:
this is unused

Reply:
Removed in 45dde003b4a796a08630d6e845ebea6b17cd279f